src/Compute/Compute/ChangeLog.md

```diff
@@ -20,6 +20,7 @@
 -->
 ## Upcoming Release
 * Fix issue with path resolution in Get-AzVmBootDiagnosticsData
+* Update Compute client library to 25.0.0.
```

Comment:
@hyonholee I would call out here (and the [`upcoming-breaking-changes.md`](https://github.com/Azure/azure-powershell/blob/master/src/Compute/documentation/upcoming-breaking-changes.md) file for Compute) the new property being added and how it's replacing `EncryptionSettings`